client/scala/examples/README.md

```diff
@@ -0,0 +1,3 @@
+To build the example Armada client programs, copy the jar file that was generated (e.g.
+scala-armada-client_2.13-0.1.0-SNAPSHOT.jar) from ../scala-armada-client/target/scala-2.13/
+into the `lib` directory, then run `sbt compile` here.
```

Comment:
As @EnricoMi mentioned in slack:
```
If you do

"sbt publishM2"

in client/scala/scala-armada-client, then you can reference the jar as a normal dependency in client/scala/examples, no need to copy the jar into lib.
```

Reply:
Fixed in commit b64208ff - thanks to @EnricoMi